cmd/find.go

```diff
@@ -190,142 +199,127 @@ func doFind(clnt Client, ctx *cli.Context) {
-			doFindWatch(ctx.Args().Get(0), ctx)
+			doFindWatch(ctx, ctx.Args().Get(0))
 		}
 	}
-
+	return nil
 }
 
-// SubArgsHelper formats the string to remove {} and replace each with the appropriate argument
-func SubArgsHelper(args, path string, fileContent contentMessage) string {
-
+// stringsReplace - formats the string to remove {} and replace each
+// with the appropriate argument
+func stringsReplace(args string, fileContent contentMessage) string {
 	// replace all instances of {}
 	str := args
 	if strings.Contains(str, "{}") {
-		str = strings.Replace(str, "{}", path, -1)
+		str = strings.Replace(str, "{}", fileContent.Key, -1)
+	}
+
+	// replace all instances of {""}
+	if strings.Contains(str, `{""}`) {
+		str = strings.Replace(str, `{""}`, strconv.Quote(fileContent.Key), -1)
 	}
 
 	// replace all instances of {base}
 	if strings.Contains(str, "{base}") {
-		str = strings.Replace(str, "{base}", filepath.Base(path), -1)
+		str = strings.Replace(str, "{base}", filepath.Base(fileContent.Key), -1)
+	}
+
+	// replace all instances of {"base"}
+	if strings.Contains(str, `{"base"}`) {
+		str = strings.Replace(str, `{"base"}`, strconv.Quote(filepath.Base(fileContent.Key)), -1)
 	}
 
 	// replace all instances of {dir}
 	if strings.Contains(str, "{dir}") {
-		str = strings.Replace(str, "{dir}", filepath.Dir(path), -1)
+		str = strings.Replace(str, "{dir}", filepath.Dir(fileContent.Key), -1)
+	}
+
+	// replace all instances of {"dir"}
+	if strings.Contains(str, `{"dir"}`) {
+		str = strings.Replace(str, `{"dir"}`, strconv.Quote(filepath.Dir(fileContent.Key)), -1)
 	}
 
 	// replace all instances of {size}
 	if strings.Contains(str, "{size}") {
-		s := humanize.IBytes(uint64(fileContent.Size))
-		str = strings.Replace(str, "{size}", s, -1)
+		str = strings.Replace(str, "{size}", humanize.IBytes(uint64(fileContent.Size)), -1)
 	}
 
-	if strings.Contains(str, "{url}") {
-		s := GetPurl(path)
-		str = strings.Replace(str, "{url}", s, -1)
+	// replace all instances of {"size"}
+	if strings.Contains(str, `{"size"}`) {
+		str = strings.Replace(str, `{"size"}`, strconv.Quote(humanize.IBytes(uint64(fileContent.Size))), -1)
 	}
 
+	// replace all instances of {time}
 	if strings.Contains(str, "{time}") {
-		t := fileContent.Time.String()
-		str = strings.Replace(str, "{time}", t, -1)
-	}
-
-	// replace all instances of {""}
-	if strings.Contains(str, "{\""+"\"}") {
-		str = strings.Replace(str, "{\""+"\"}", strconv.Quote(path), -1)
-	}
-
-	// replace all instances of {"base"}
-	if strings.Contains(str, "{\""+"base"+"\"}") {
-		str = strings.Replace(str, "{\""+"base"+"\"}", strconv.Quote(filepath.Base(path)), -1)
+		str = strings.Replace(str, "{time}", fileContent.Time.Format(printDate), -1)
 	}
 
-	// replace all instances of {"dir"}
-	if strings.Contains(str, "{\""+"dir"+"\"}") {
-		str = strings.Replace(str, "{\""+"dir"+"\"}", strconv.Quote(filepath.Dir(path)), -1)
+	// replace all instances of {"time"}
+	if strings.Contains(str, `{"time"}`) {
```

Comment:
{"time"} check is also duplicated. The if block above at line#296 needs to be removed.